DuckDuckGo/SettingsViewController.swift

```diff
@@ -127,7 +127,8 @@ class SettingsViewController: UITableViewController {
     private lazy var shouldShowNetPCell: Bool = {
 #if NETWORK_PROTECTION
         if #available(iOS 15, *) {
-            return featureFlagger.isFeatureOn(.networkProtection)
+            let accessController = NetworkProtectionAccessController()
```

Comment:
Heads up @afterxleep

Reply:
oh!, no more feature flagging @graeme?  Here's where this logic has moved:

https://github.com/duckduckgo/iOS/pull/2255/files#diff-ad06089c79361f33a5f83335daca81941fcb6ec2fda955c52a8783b1a551ba61R49

Should we kill the feature flag check from there?